### PR TITLE
Update sophos alerts

### PR DESCRIPTION
**Pull request by roaaattalla**

### CODE REVIEW

File: cyences_app_for_splunk/default/data/ui/views/cs_sophos_reports.xml [hunk 1]
```diff
@@ -227,7 +227,7 @@
       <title>Malware Detected</title>
       <table>
         <search>
-          <query>cs_sophos` sourcetype="sophos_events" type="Event::Endpoint::CoreDetection" $tkn_host$ $tkn_dhost$ $tkn_user$
+          <query> cs_sophos` sourcetype="sophos_events" type="Event::Endpoint::CoreDetection" $tkn_host$ $tkn_dhost$ $tkn_user$
```
Comment (roaaattalla) [inline cyences_app_for_splunk/default/data/ui/views/cs_sophos_reports.xml]: Already changed

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -2959,30 +2959,30 @@ display.page.search.tab = statistics
...
 | `cs_human_readable_time_format(_time, event_time)` \
 | `cs_sophos_endpoint_not_protected_filter`
 action.cyences_notable_event_action = 1
 action.cyences_notable_event_action.param.filter_macro_name = cs_sophos_endpoint_not_protected_filter
-action.cyences_notable_event_action.contributing_events = `cs_sophos` type="Event::Endpoint::NotProtected"
-action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dhost
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` dhost=$row.dhost$
+action.cyences_notable_event_action.contributing_events = `cs_sophos` sourcetype="sophos_events" type="Event::Endpoint::NotProtected"
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by location
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` location=$row.location$
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:2985]: Add the sourcetype="sophos_events" filter

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -3011,23 +3011,23 @@ request.ui_dispatch_app = cyences_app_for_splunk
...
-| `cs_sophos_realtime_protection_disabled_filter`
+| `cs_sophos_realtime_protection_disabled_filter` 
 action.cyences_notable_event_action = 1
 action.cyences_notable_event_action.param.filter_macro_name = cs_sophos_realtime_protection_disabled_filter
-action.cyences_notable_event_action.contributing_events = `cs_sophos` type="Event::Endpoint::SavDisabled"
-action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dhost
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` dhost=$row.dhost$
+action.cyences_notable_event_action.contributing_events = `cs_sophos` sourcetype="sophos_events" type="Event::Endpoint::SavDisabled"
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by location
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` location=$row.location$
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3030]: Add the sourcetype="sophos_events" filter

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 4]
```diff
@@ -3054,30 +3054,30 @@ display.page.search.tab = statistics
...
 | `cs_human_readable_time_format(_time, event_time)` \
 | `cs_sophos_service_not_running_filter`
 action.cyences_notable_event_action = 1
 action.cyences_notable_event_action.param.filter_macro_name = cs_sophos_service_not_running_filter
-action.cyences_notable_event_action.contributing_events = `cs_sophos` type="Event::Endpoint::ServiceNotRunning"
-action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dhost
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` dhost=$row.dhost$
+action.cyences_notable_event_action.contributing_events = `cs_sophos` sourcetype="sophos_events" type="Event::Endpoint::ServiceNotRunning"
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by location
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` location=$row.location$
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3080]: Add the sourcetype="sophos_events" filter

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 5]
```diff
@@ -3102,16 +3102,16 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_sophos` type IN ("Event::Endpoint::Threat::CleanupFailed", "Event::Endpoint::CoreCleanFailed", "Event::Endpoint::CoreHmpaCleanFailed", "Event::Endpoint::CoreSystemCleanFailed") \
-| stats count, latest(_time) as _time, values(name) as threat, values(src_ip) as src_ip, values(suser) as user by host, dhost | sort -count \
+`cs_sophos` sourcetype="sophos_events" type IN ("Event::Endpoint::Threat::CleanupFailed", "Event::Endpoint::CoreCleanFailed", "Event::Endpoint::CoreHmpaCleanFailed", "Event::Endpoint::CoreSystemCleanFailed") \
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3105]: **search =** key is missing.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 6]
```diff
@@ -3102,16 +3102,16 @@ display.page.search.tab = statistics
...
+| stats count, latest(_time) as _time, values(name) as threat, values(source_info.ip) as src_ip, values(source) as user by host, location | sort -count \
 | eval cyences_severity = "critical" \
 | `cs_human_readable_time_format(_time, event_time)` \
-| `cs_sophos_failed_to_cleanup_threat_filter`
+| `cs_sophos_failed_to_cleanup_threat_filter` \
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3109]: Remove the forward slash (\) at the end.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 7]
```diff
@@ -3102,16 +3102,16 @@ display.page.search.tab = statistics
...
 action.cyences_notable_event_action = 1
 action.cyences_notable_event_action.param.filter_macro_name = cs_sophos_failed_to_cleanup_threat_filter
-action.cyences_notable_event_action.contributing_events = `cs_sophos` type IN ("Event::Endpoint::Threat::CleanupFailed", "Event::Endpoint::CoreCleanFailed", "Event::Endpoint::CoreHmpaCleanFailed", "Event::Endpoint::CoreSystemCleanFailed")
-action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dhost
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` dhost=$row.dhost$
+action.cyences_notable_event_action.contributing_events = `cs_sophos` "sophos_events" type IN ("Event::Endpoint::Threat::CleanupFailed", "Event::Endpoint::CoreCleanFailed", "Event::Endpoint::CoreHmpaCleanFailed", "Event::Endpoint::CoreSystemCleanFailed")
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3112]: Add the **sourcetype=**"sophos_events" filter

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 8]
```diff
@@ -3102,16 +3102,16 @@ display.page.search.tab = statistics
...
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by location
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_sophos` location=$row.location$
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:3114]: Add the sourcetype="sophos_events" filter

**Review by hardikhdholariya — CHANGES_REQUESTED** [review]

**Review by hardikhdholariya — APPROVED** [review]

**Pull request merged**